webknossos/tests/test_dataset_deprecated.py

```diff
@@ -153,7 +153,7 @@ def assure_exported_properties(ds: Dataset) -> None:
 def test_create_dataset_with_layer_and_mag() -> None:
     rmtree(TESTOUTPUT_DIR / "wkw_dataset")
 
-    ds = Dataset(TESTOUTPUT_DIR / "wkw_dataset", scale=(1, 1, 1))
```

Comment:
This is the test file for deprecated usages. I think we should keep `scale` here.